Sources/SentryCrash/Installations/SentryCrashInstallation.h

```diff
@@ -43,12 +43,13 @@
  * WARNING: Only call async-safe functions from this function! DO NOT call
  * Objective-C methods!!!
  */
-@property (atomic, readwrite, assign) SentryCrashReportWriteCallback onCrash;
+@property (nullable, atomic, readwrite, assign) SentryCrashReportWriteCallback onCrash;
 
 /** Install this installation. Call this instead of -[SentryCrash install] to
  * install with everything needed for your particular backend.
+ * If you wish to use default cache directory, pass @c nil .
  */
-- (void)install;
+- (void)install:(nullable NSString *)customCacheDirectory;
```

Comment:
`m`: We only call this from `SentryCrashIntegration` where we can access the options, so I would prefer making this nonnullable so you always have to pass it in.
```suggestion
- (void)install:(NSString *)customCacheDirectory;
```